templates/main.php

```diff
@@ -31,13 +31,11 @@ class="collection"
                     </span>
                     <span class="title">{{ collection.displayname }}</span>
                 </a>
-                <div class="app-navigation-entry-utils">
-                    <ul>
-                        <li class="app-navigation-entry-utils-counter">{{ getCollectionString(collection.id) | counterFormatter }}</li>
-                    </ul>
+                <div class="app-navigation-entry-utils-counter">
```

Comment:
You need the   ` <div class="app-navigation-entry-utils">` to fit the css guidelines.
https://docs.nextcloud.com/server/11/developer_manual/app/css.html#menus